### DISCUSSION TITLE
Foocus 2-5-0 Doesn't work and start

### DISCUSSION THREAD

**Discussion by DarkRuler1981** [lllyasviel/Fooocus#3558] [General]
Hello, my name is Chris, and this is my first post.
I hope I don't start causing trouble right away.

I downloaded and installed Foocus 2-5-0 for Win XP.

It was working, but suddenly after a Windows update, it stopped launching, and I don't understand why.
What information do you need to figure out what happened?

Thanks to everyone

---
My Req:
- nvidia 1660
- cpu 12th Gen Intel(R) Core(TM) i9-12900KF   3.19 GHz
-32 gB ram DDR 5

**Comment by Rogala** [comment]
I'm even afraid to ask, do you have windows xp installed on this hardware?

**Reply by IPv6** [reply]
> after a Windows update

there was also an update! sounds like horror movie now

**Comment by IPv6** [comment]
You can post terminal content, if the problem have obvious solution - will be able to help.

But honestly this is unexpected to work on WinXP. embedded python certainly did not suitable for it, and much less chances pytorch will works as expected.

**Comment by DarkRuler1981** [comment]
I have Win XP 

so this is screen when i use RUN-Realistic

![1_0_FoocusAI_RunRealistic](https://github.com/user-attachments/assets/ea95a169-b996-4d1b-98d0-5e0ab515ac24)

**Reply by Rogala** [reply]
I don't know where you could find drivers for your hardware that came out later than the end of Windows xp support, you should [definitely go here](https://www.microsoft.com/en-us/software-download/windows10).
